src/js/common/schemaform/widgets/ArrayCountWidget.jsx

```diff
@@ -1,45 +1,45 @@
 import React from 'react';
 
-// We're expanding or contracting the array based on the count
-// and returning undefined if the array should be empty
-function getValue(count, value = []) {
-  if (!count || parseInt(count, 10) < 1) {
-    return undefined;
+export default class SpouseMarriageWidget extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { value: !!props.value
+      ? props.value.length - (this.props.options.countOffset || 0)
```

Comment:
When would the zero be used? Is countOffset not always set at -1 for the spouse?